build.gradle

```diff
@@ -2,8 +2,13 @@ plugins {
     // to check for updates run: ./gradlew dependencyUpdates -Drevision=release
     id 'com.github.ben-manes.versions' version '0.50.0'
 }
-ext.currentVersion = '0.9.3-SNAPSHOT'
-ext.lastRelease = '0.9.2'
+
+ext {
+    currentVersion = '0.9.3-SNAPSHOT'
+    lastRelease = '0.9.2'
+    junitVersion = '5.12.0'
```

Comment:
I like the idea of using variables for these common version updates, but do you know if dependabot will be able to track down the source and open up a pull request for this?